File: mario_less/mario.c
#include <cs50.h>
#include <stdio.h>

// Prompt user for height correct
int get_height(void);

int main(void) {
    int height = get_height();

    for (int i = 0; i < height; i++) {
        for (int l = 0; l < height - i; l++) {
            printf(" ");
        }

        for (int j = 0; j <= i; j++) {
            printf("#");
        }

        printf("\n");
    }
}

int get_height(void) {
    int height = get_int("Height: ");

    // loop for handling error
    while (height <= 0 || height > 8) {
        printf("Error!, Please enter a number between 1 and 8!\n");
        height = get_int("Height: ");
    }

    return height;
}

File: Python/mario_more/mario.py
from cs50 import get_int

def get_height():
    height = 0
    while height <= 0 or height > 8:
        height = get_int("Height: ")

    return height


height = get_height()

for i in range(0, height):
    for l in range(1, height - i):
        print(" ", end="")

    for j in range(0, i +1):
        print("#", end="")

    print("  ", end="")

    for j in range(0, i +1):
        print("#", end="")
    print()

